exercises/beer-song/test/beer_song_test.exs

```diff
@@ -1,356 +1,381 @@
 defmodule BeerSongTest do
   use ExUnit.Case
 
-  test "getting the first verse (99 bottles)" do
-    assert BeerSong.verse(99) == """
-           99 bottles of beer on the wall, 99 bottles of beer.
-           Take one down and pass it around, 98 bottles of beer on the wall.
-           """
+  describe "verse" do
+    describe "single verse" do
```

Comment:
Nested describes are not supported in Elixir, so you'll need to find another way to set this up.

Reply:
Thanks for pointing out! I think just dropping the second level is okay.